roles/ceph-rbd-mirror/tasks/start_rbd_mirror.yml

```diff
@@ -1,7 +1,14 @@
 ---
-- name: start and add that the rbd mirror service to the init sequence
+- name: stop and remove the generic rbd-mirror service instance
   service:
       name: "ceph-rbd-mirror@{{ ceph_rbd_mirror_local_user }}"
+      status: stopped
+      enabled: no
+  changed_when: false
```

Comment:
We need an `ignore_error` on this on since some deployment (newer) won't have it.